### DISCUSSION TITLE
my solidity journey

### DISCUSSION THREAD

**Discussion by Okhayeeli** [Cyfrin/foundry-full-course-cu#1475] [General]
i finished the first part of the lesson today, wish me luck as I move forward

**Comment by EngrPips** [comment]
Hello @Okhayeeli , I wish you the best in your journey

**Comment by its0Vivek** [comment]
Let's embark on a Solidity journey together!

**Comment by saint-gmoney0** [comment]
hi everyone am new to web3 development hope am welcome
@PatrickAlphaC

**Comment by EngrPips** [comment]
Welcome onboard
